vscode/microsoft-kiota/src/commands/generate/generateClientCommand.ts

```diff
@@ -131,6 +138,8 @@ export class GenerateClientCommand extends Command {
 
       const pathOfSpec = path.join(outputPath, `${outputState.clientClassName?.toLowerCase()}-openapi.yml`);
       const pathPluginManifest = path.join(outputPath, `${outputState.clientClassName?.toLowerCase()}-apiplugin.json`);
+      const apiManifestPath = path.join(outputPath, ".kiota", API_MANIFEST_FILE);
```

Comment:
It would be beneficial to extract the repeated `.kiota` string into a constant

Reply:
Nice catch. These additions will no longer be necessary as the team said they'd infer the location of the .kiota folder. I will be deleting the lines adding .kiota